_includes/footer.html

```diff
@@ -1,4 +1,4 @@
-<footer class="footer" role="contentinfo">
+<footer class="footer">
```

Comment:
I know some automated a11y checkers flag `role="contentinfo"` as being redundant, but I'd recommend we keep this in because it's still useful for some older browsers and doesn't have any negative impacts.